mypy/build.py

```diff
@@ -544,6 +548,23 @@ def find_config_file_line_number(path: str, section: str, setting_name: str) ->
     return -1
 
 
+class FindModuleCache:
```

Comment:
Add docstring. Mention that this never flushes the FileSystemCache and caller is responsible for that.